lib/Person.php

```diff
@@ -14,7 +14,6 @@
  * @property mixed $address_kana
  * @property mixed $address_kanji
  * @property int $created
- * @property bool $deleted
```

Comment:
please leave this one. A Person can be deleted and the returned resource would have `deleted: true`